README.md

```diff
@@ -0,0 +1,67 @@
+CMSIS-SVD Repository
+====================
+
+This repository provides a convenient place to access and aggregate CMSIS-SVD hardware descriptions from multiple sources.
+
+What is CMSIS-SVD
+-----------------
+
+ARM provides the following description of
+[CMSIS-SVD (System View Description)](http://www.keil.com/pack/doc/CMSIS/SVD/html/index.html)
+
+> The CMSIS System View Description format(CMSIS-SVD) formalizes the
+> description of the system contained in ARM Cortex-M processor-based
+> microcontrollers, in particular, the memory-mapped registers of
+> peripherals. The detail contained in system view descriptions is
+> comparable to the data in device reference manuals. The information
+> ranges from high-level functional descriptions of a peripheral all the
+> way down to the definition and purpose of an individual bit field in a
+> memory-mapped register.
+
+The original vision of ARM appears to have been to aggregate SVDs from
+various sources into a single repository accessible via
+[ARM's CMSIS Website](http://www.arm.com/products/processors/cortex-m/cortex-microcontroller-software-interface-standard.php).
+Currently, however, this site fails to provide a comprehensive
+repository of CMSIS SVDs.  The SVDs in this repository have been
+previously aggregated as part of the Eclipse
+[Embedded Systems Register View](http://embsysregview.sourceforge.net/)
+plugin.  This repo builds on the shoulders of the developers of
+EmbSysRegView and seeks to provide a repository of SVDs that is more
+accessible to a greater number of other projects.
+
+How Can the CMSIS-SVD Be Used
+-----------------------------
+
+The generic description of each MCUs CPU and hardware registers is
+very valuable when generating code that can be used for talking to
+specific target hardware.  In fact, much of the code in parts of CMSIS
+itself are generated based on the SVD.  ARM distributes an executable
+that does this transformation (SVDConvert.exe).
+
+The information can also be used for building debug tooling, test
+infrastructure, or whatever else.
+
+Contributing
+------------
+
+Contributions from Silicon Vendors, as well as community members, are
+greatly appreciated.  Please feel free to create an issue on Github
+and/or submit Pull Requests with proposed changes to the repository.
+These will be reviewed and pulled in if deemed appropriate.
```

Comment:
As it relates to contributions, especially with license considerations and avoiding changes, a CONTRIBUTING.md probably makes sense.  I think we could also consider pointing people toward something like https://github.com/rust-embedded/svdtools

I could see this repo potentially also be a place to manage svdtool patches.  I haven't used it significantly but am aware of it as a member of the rust-embedded wg.

Reply:
@posborne ok for me, this is a good idea. I have add a CONTRIBUTING.md file which specify that any patch must follow the rust-embedded/svdtools patch format.  And in the README, that this repository aggregates rust-embedded/svdtools patchs.